svc-systems/README.md

```diff
@@ -977,6 +976,8 @@ curl -i -X DELETE \
 |@Redfish.OperationApplyTimeSupport|Redfish annotation \(optional\)<br> | It enables you to control when the operation is carried out.<br> Supported value is: `OnReset`. Supported values are: `OnReset` and `Immediate`. `OnReset` indicates that the volume will be deleted only after you reset the system.<br> |
 
 
+<br>**NOTE:**<br> Deleted volume will be reflected only after the system resets. To see the change, [reset the computer system](#resetting-a-computer-system).
```

Comment:
individual addition/deletion will be triggered after reboot. In the meanwhile we cannot add or delete volumes.

Reply:
updated